cmd/argocd/commands/initialize/cmd.go

```diff
@@ -0,0 +1,32 @@
+package initialize
```

Comment:
I think InitCommand is only used by admin/dashboard, so it could be a private function there. As for RetrieveContextIfChanged, maybe a better place for it. I think with those two changes the initialize package can be removed entirely?

Reply:
Actually, InitCommand is still used where ever it was previously because it did an extra step apart from just starting the server. It took all the kubeconfig flags and put them into the cmd context. I kept that functionality intact because if the cmd context is accessed in the command for some reason apart from the API server then it will break the code. If the context is not needed anywhere I can shift that part too and make this function private